app/src/main/java/nextstep/payments/ui/main/PopulatedPaymentCardScreen.kt

```diff
@@ -15,26 +16,32 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.luminance
+import androidx.compose.ui.graphics.toArgb
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import nextstep.payments.data.BankType
 import nextstep.payments.data.Card
 import nextstep.payments.ui.component.MaskedCardNumberText
 import nextstep.payments.ui.component.PaymentCard
 
 @Composable
 fun PopulatedPaymentCard(
     card: Card,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    onCardClick: () -> Unit,
```

Comment:
```suggestion
    onCardClick: () -> Unit,
    modifier: Modifier = Modifier,
```
파라미터 순서는 필수 > Optional 이므로, modifier가 아래에 위치해야합니다.

Reply:
아 이 컨벤션을 유지를 해야 되는 거군요 감사합니다!